src/api/libcellml/units.h

```diff
@@ -423,7 +431,7 @@ class LIBCELLML_EXPORT Units: public NamedEntity, public ImportedEntity
      * @return @c true when these @c Units rely on @c Units which are imported,
      * or @c false otherwise.
      */
-    bool requiresImports() const;
```

Comment:
Did we have to drop the const here?

Reply:
Yes, because `Units::requiresImports()` now calls `shared_from_this()`.